### PR TITLE
Always use base64 for attachments

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

File: sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java [hunk 1]
```diff
@@ -46,19 +44,14 @@
...
-    private String contentType;
-
-    /**
      * File name.
      */
     private String fileName;
 
     /**
-     * Data (plain text or base64 string for binary data).
+     * Data encoded as base64.
```
Comment (MatkovIvan) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java]: Data not encoded as base64 here. You are store data as is and encode only for json serialize.

File: versions.gradle [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ext {
     versionCode = 19
-    versionName = '0.8.1'
+    versionName = '0.9.0'
```
Comment (jaelim-ms) [inline versions.gradle]: I think 0.8.2 should be enough

Comment (guperrot) [inline versions.gradle]: No, error attachments is a feature

File: sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java (hunk 1 above)
Comment (guperrot) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java]: This is just the description we copy from swagger though.

Comment (guperrot) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java]: adapted comment

File: sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java [hunk 3]
```diff
@@ -68,52 +74,25 @@
...
-
-    /**
-     * Build an error attachment log with text and binary suitable for using in {link CrashesListener#getErrorAttachments(ErrorReport)}.
-     *
      * @param data        binary data.
-     * @param text        text to attach to the attachment log.
      * @param fileName    file name to use in error attachment log.
      * @param contentType binary data MIME type.
-     * @return ErrorAttachmentLog attachment or null if text and data are null.
+     * @return ErrorAttachmentLog attachment or null if null data is passed.
```
Comment (jaelim-ms) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java]: `if data is null.` instead?

Comment (guperrot) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java]: oops, its never null actually, old comment

**Pull request merged**